src/lib/utils/getEndpoint.ts

```diff
@@ -0,0 +1,33 @@
+export type BackendDestination = 'PORTAL_ENDPOINT' | 'SWC_ENDPOINT'
+// These variables indicate which endpoint should be called
+export const REPO_ENDPOINT: BackendDestination = 'PORTAL_ENDPOINT'
+export const PORTAL_ENDPOINT: BackendDestination = 'SWC_ENDPOINT'
+
+// Encodes the portal and repo service destinations
+type EndpointObject = {
+  REPO: string
+  PORTAL: string
+}
+
+export const PRODUCTION_ENDPOINT_CONFIG: EndpointObject = {
```

Comment:
I wonder if these objects can just be distilled to an ENUM keyed off the BackendDestination  (may be the answer is 'no':)